installers/osie/main.go

```diff
@@ -81,6 +81,10 @@ func kernelParams(action, state string, j job.Job, s *ipxe.Script) {
 		s.Args("grpc_cert_url=" + grpcCertURL)
 		s.Args("registry_username=" + registryUsername)
 		s.Args("registry_password=" + registryPassword)
+		s.Args("log_driver=" + log_driver)
+		s.Args("log_tag=" + log_tag)
+		s.Args("log_server_address=" + log_server_address)
+		s.Args("log_server_address_type=" + log_server_address_type)
```

Comment:
I'm not sure what log_server_address_type means, tcp, udp, ...?

Reply:
@mmlb as mentioned in the above description, this PR needs to be reviewed with Tink and Osie Pr.
https://github.com/tinkerbell/tink/pull/267
https://github.com/tinkerbell/osie/pull/104

Reply:
details of the variable are present here https://github.com/tinkerbell/tink/pull/267/files#diff-95eba583451d93c0a847d4e39967c981

Reply:
Gotcha. This info should still be made available in this commit, via the commit message. Ditto for the other PRs.

I'd like for the options to mimic the docker cmdline flags, this helps keeping context of what some variable maps to. Why not `log_opt_syslog-address=log_server_address` instead? Then osie can handle `log_opt*` commands more generically.

Reply:
IMO having `syslog` added in env would be incorrect. To show env is part of `log-opt` we can add `opt` in env variables. 
like `LOG_OPT_SERVER_ADDRESS_TYPE`.